src/dataTables.altEditor.free.js

```diff
@@ -645,13 +645,15 @@
                                 // array-style select or select2
                                 for (var i = 0; i < optionsArray.length; i++) {
                                     options += "<option value='" + this._quoteattr(optionsArray[i])
+									+ "'"+(optionsArray[i]===columnDefs[j].value?" selected":"") //enables default value for select
                                         + "'>" + optionsArray[i] + "</option>";
                                 }
                             } else {
                                 // object-style select or select2
                                 for (var x in optionsArray) {
-                                    options += "<option value='" + this._quoteattr(x) + "' >"
-                                        + optionsArray[x] + "</option>";
+                                    options += "<option value='" + this._quoteattr(x) + "'"
```

Comment:
I don't think this is the right approach. May have undesired effects. Values should not initialized in the creatDialog() functions, they should be set later, e.g. here:

https://github.com/KasperOlesen/DataTable-AltEditor/blob/4d1728fd8c293a4f239243d4803a88b3e5b7e4ea/src/dataTables.altEditor.free.js#L386